cyences_app_for_splunk/bin/user_inventory_util.py

```diff
@@ -210,7 +210,8 @@ def get_as_dict(self, user):
             'user_types': list(user.get("user_types").keys()),
             'indexes': list(user.get("indexes").keys()),
             'sourcetypes': list(user.get("sourcetypes").keys()),
-            'users': list(user.get("users").keys())
+            'users': list(user.get("users").keys()),
+            'privilege_user_role': str(user.get("privilege_user_role", "-"))
```

Comment:
call it **is_previleged_user**, to make it more like a boolean field in the User inventory as well in the dashboard.

Reply:
FYI, this field will display the role name which user has provided. are you expecting the yes or no on UI?